config/application.rb

```diff
@@ -86,6 +86,11 @@ class Application < Rails::Application
     }
 
     config.action_controller.action_on_unpermitted_parameters = :raise
+    
+    config.active_job.queue_adapter = :sucker_punch
+    config.samson.export_job = ActiveSupport::OrderedOptions.new
+    config.samson.export_job.downloaded_age = ENV['EXPORT_JOB_DOWNLOADED_AGE'] || 12.hours
```

Comment:
env variables are strings ... so need to convert to integers or math above will blow up

`(ENV['EXPORT_JOB_DOWNLOADED_AGE'] || 12.hours).to_i`